Update/haji_002.txt

```diff
@@ -49,313 +49,313 @@ void main()
 	OutputLine(NULL, "あんまり人が多いようだったら自重しようかね。",
-			NULL, "", Line_WaitForInput);
+			NULL, "We'll have to restrain ourselves in front of others.", Line_WaitForInput);
 	PlaySE(4, "ps3/s23/03/vth_mion1016", 256, 64);
 	OutputLine(NULL, "行ってから考えてもいいでしょ」",
-			NULL, "", Line_Normal);
+			NULL, "Let's cross that bridge when we get there.", Line_Normal);
 	ClearMessage();
 
 //レナrvS23/02/VTH_rena1009.「そうだね。kvS23/02/VTH_rena1010.レナは部活抜きで、のんびりプールに浸かるのでも十分楽しいかな、かな」
 	PlaySE(4, "ps3/s23/02/vth_rena1009", 256, 64);
 	OutputLine(NULL, "「そうだね。",
-			NULL, "", Line_WaitForInput);
+			NULL, "Yeah...", Line_WaitForInput);
 	PlaySE(4, "ps3/s23/02/vth_rena1010", 256, 64);
 	OutputLine(NULL, "レナは部活抜きで、のんびりプールに浸かるのでも十分楽しいかな、かな」",
-			NULL, "", Line_Normal);
+			NULL, "Rena thinks a relaxing day at the pool is plenty fun, yeah. Yeah.", Line_Normal);
 	ClearMessage();
 
 	DrawBustshot( 5, "sprite/normal/ri1_niko_a1_0", -160, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, 18, 200, TRUE );
 
 //梨花rvS23/05/VTH_rika1008.「……みー。ボクもそれで十分楽しいのですよ」
 	PlaySE(4, "ps3/s23/05/vth_rika1008", 256, 64);
 	OutputLine(NULL, "「……みー。ボクもそれで十分楽しいのですよ」",
-			NULL, "", Line_Normal);
+			NULL, "Meep. I think it sounds fun too.", Line_Normal);
 	ClearMessage();
 
 	DrawBustshot( 4, "sprite/normal/me1b_def_a1_0", 0, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, 19, 200, TRUE );
 
 //魅音rvS23/03/VTH_mion1017.「はは、たまにはそういうのもいいかもねぇ。kvS23/03/VTH_mion1018.ま、日々の激戦を忘れて、のんびり過ごすのもいいでしょ」
 	PlaySE(4, "ps3/s23/03/vth_mion1017", 256, 64);
 	OutputLine(NULL, "「はは、たまにはそういうのもいいかもねぇ。",
-			NULL, "", Line_WaitForInput);
+			NULL, "Haha, certainly days like that are good every once in a while.", Line_WaitForInput);
 	PlaySE(4, "ps3/s23/03/vth_mion1018", 256, 64);
 	OutputLine(NULL, "ま、日々の激戦を忘れて、のんびり過ごすのもいいでしょ」",
-			NULL, "", Line_Normal);
+			NULL, "Well, setting aside our daily battles and just relaxing sounds plenty fine.", Line_Normal);
 	ClearMessage();
 
 //圭一rvS23/01/VTH_k11013.「へ～、珍しいな。kvS23/01/VTH_k11014.俺はてっきり罰ゲームに、とんでもないものを用意してるんじゃないかって思ったけどな」
 	PlaySE(4, "ps3/s23/01/vth_k11013", 256, 64);
 	OutputLine(NULL, "「へ～、珍しいな。",
-			NULL, "", Line_WaitForInput);
+			NULL, "Oh, that's unusual.", Line_WaitForInput);
 	PlaySE(4, "ps3/s23/01/vth_k11014", 256, 64);
 	OutputLine(NULL, "俺はてっきり罰ゲームに、とんでもないものを用意してるんじゃないかって思ったけどな」",
-			NULL, "", Line_Normal);
+			NULL, "I was positive you already had a penalty game planned ahead of time.", Line_Normal);
 	ClearMessage();
 
 //r…珍しく、部活メンバーでのんびりと過ごすことができそうだな。
 	OutputLine(NULL, "…珍しく、部活メンバーでのんびりと過ごすことができそうだな。",
-			NULL, "", Line_Normal);
+			NULL, "...It seems like everyone will get to relax for once.", Line_Normal);
 	ClearMessage();
 
 //r…もっとも油断はできんが。
 	OutputLine(NULL, "…もっとも油断はできんが。",
-			NULL, "", Line_Normal);
+			NULL, "...Though that's how it should be.", Line_Normal);
 	ClearMessage();
 
 //rどうせ当日は、何だかんだと大暴れになるに違いない。
 	OutputLine(NULL, "どうせ当日は、何だかんだと大暴れになるに違いない。",
-			NULL, "", Line_Normal);
+			NULL, "At any rate, we'll get to go crazy on Sunday.", Line_Normal);
```

Comment:
Going with the above, I think this was supposed to be like "well when the day comes I'm sure we'll end up doing something crazy"